docs/docs-beta/docs/dagster-plus/saved-views.md

```diff
@@ -0,0 +1,62 @@
+
+</details>
+
+
+## Tailored catalog views
+
+Catalog views allow you to toggle between sets of assets that you care about most. Here are a few examples of how your team might use catalog views:
+
+- **Team view**: A view that filters assets based on ownership to show only the assets owned by your team.
+- **Ingestion view**: A view that filters assets based on the asset kind to give insight into the status of your ELT ingestion.
+- **Gold medallion view**: A view which displays assets that have a "gold" medallion tag, showing only refined, high-quality data that analysts can use with confidence.
+
+
+## Create catalog views
+
+In any Dagster+ catalog page, you can access the current catalog view, or create a new catalog view with the catalog view button on the top left of the screen. By default, this button is labeled "All assets", and has a globe icon.
+
+To create a new catalog view, you have two options:
+1. Create a new catalog view from scratch, from the catalog view menu.
+2. Create a new catalog view from your current set of filters.
+
+### Create a new catalog view from scratch
+
+To build a new catalog view, click the catalog view button to open the catalog view menu. From here, click the "New" button.
+
+Give your view a name and optionally a description and icon. Next, you can select one or more filters to apply to your view by clicking the "Add filters" button. Filters can select a subset of assets based on their metadata, tags, kinds, owners, asset groups, or other properties.
+
+Finally, you can choose whether to share your view with your team, or keep it private to yourself by toggling the "Public view" switch.
+
+To finish, click "Create view" to finalize your new catalog view.
+
+### Create a new catalog view from your current set of filters
+
+When viewing the global asset lineage or asset list, you can create a new catalog view from your current set of filters. First, select one or more asset filters. Then, on the top right of the screen, click the "Create new catalog view" button. This will open the catalog view creation dialog with your current filters pre-populated.
+
+From here, give your view a name and optionally a description and icon. You can also choose whether to share your view with your team, or keep it private to yourself by toggling the "Public view" switch.
+
+To finish, click "Create view" to finalize your new catalog view.
+
+## Edit, duplicate, or delete catalog views
+
+To edit, duplicate, or delete a catalog view, navigate to the catalog view menu by clicking the catalog view button on the top left of the screen. From here, you can search for the view you want to modify, duplicate, or delete. Use the three-dot menu on the right side of the view to show the available actions.
+
+When editing a catalog view, any active filters will automatically be included in the set of changes. You can modify the view's name, description, icon, and sharing settings. To save your changes, click "Save changes".
```

Comment:
I think this could also benefit from being in a list:

```suggestion
1. Click the catalog view button to open the catalog view menu.
2. Search for the view you want to edit, duplicate, or delete.
3. Click the **three dot menu** to the right of the view to display available options.
4. If modifying the view, note that any active filters will automatically be included in the set of changes. You can also change the view's name, description, icon, and sharing settings. 5. When finished, click **Save changes**.
```